### PR TITLE
NEON AOP Catalog Updates Aug 2024

### PR DESCRIPTION
**Pull request by bhass-neon**
Updates to add links to the tutorials to the descriptions, add comments to example scripts, and fix some mistakes in the catalogs. Had to remove the skip_indexing test in checker/node/BUILD in order to get the GitHub Actions Build step to complete.

### CODE REVIEW

File: catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_HSI_REFL_001.jsonnet [hunk 1]
```diff
@@ -24,31 +24,33 @@ local units = import 'units.libsonnet';
   title: 'NEON Directional Surface Reflectance',
   'gee:type': ee_const.gee_type.image_collection,
   description: |||
-    The NEON AOP surface directional reflectance data is an orthorectified 
-    hyperspectral VSWIR (visible to shortwave infrared) data product, containing 
+    The NEON AOP surface bidirectional reflectance data is a hyperspectral
```
Comment (simonff) [inline catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_HSI_REFL_001.jsonnet:27]: But I thought this is the directional reflectance dataset and HSI_REFL_002 is the bidirectional reflectance one, no?

Comment (bhass-neon) [reply catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_HSI_REFL_001.jsonnet]: You're correct! Just fixed in [0aab173](https://github.com/google/earthengine-catalog/pull/1111/commits/0aab1738a3c63dd20ba01becce3370e4271e564e)

File: catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_HSI_REFL_001_original.txt [hunk 2]
```diff
@@ -0,0 +1,3855 @@
+local id = 'projects/neon-prod-earthengine/assets/HSI_REFL/001';
```
Comment (simonff) [inline catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_HSI_REFL_001_original.txt]: Could you remove this file?

Comment (bhass-neon) [reply catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_HSI_REFL_001_original.txt]: I thought I did this, in [61121d6](https://github.com/google/earthengine-catalog/pull/1111/commits/61121d654953d021cc02f0dcd45d1e3e893debca). Checking.

**Review by simonff — APPROVED** [review]

**Pull request merged**